mlflow/tracking/fluent.py

```diff
@@ -150,7 +150,7 @@ def start_run(run_uuid=None, experiment_id=None, source_name=None, source_versio
     return _active_run_stack[-1]
 
 
-def end_run(status="FINISHED"):
+def end_run(status=RunStatus.to_string(RunStatus.FINISHED)):
```

Comment:
n00b questions for you: is the goal to have functions take in the enum and not the string? seems true for create_run above but not for this one (end_run). or these functions are meant for different contexts so we want the enum for one and the string for the other ?

Reply:
but we do want this `end_run` function to take in a string because it's exposed to the end user, right? (so this PR is good to go?) and eventually we want all the _internal_ methods to take in the enum.

Reply:
Yeah. I agree that user facing functions to take in string-y fields and we can check for valid values.

This change only replaces the default value of status from constant `"FINISHED"` to `RunStatus.to_string(RunStatus.FINISHED)` <-- still a string argument.